app/views/admin/loans/content/_questionnaire_group.html.slim

```diff
@@ -64,19 +70,42 @@
                 prompt: I18n.t("select_prompt"), data: {boolean: true}
 
             - if value.has_sheet?
-              - sheet = value.embeddable_media || EmbeddableMedia.new
-              .view-element
-                - if sheet.url
-                  p.spreadsheet-links
-                    a[href="#" data-expands="iframe-#{field.attribute_sym}"
-                      data-content="<iframe src='#{sheet.url}'>"]
-                      = t("loan.spreadsheet_inline")
-                    a href="#" data-hides="iframe-#{field.attribute_sym}"
-                      = t("loan.spreadsheet_hide")
-                    = ' - '
-                    = link_to t("loan.spreadsheet_new_tab"), sheet.url, {target: '_blank'}
-
-                  div.spreadsheet data-expandable="iframe-#{field.attribute_sym}"
+              - sheet = value.embeddable_media
+              .spreadsheet-links
+                - if sheet
+                  .view-element.doc-actions
+                    - if sheet.display_url.present?
+                      / Beware, the display_height value doesn't not seem to be honored
+                      a[href="#" data-expands="iframe-#{field.attribute_sym}"
+                        data-content="<iframe src='#{sheet.display_url}' height='#{sheet.display_height}'>"]
```

Comment:
Not sure why this was done as an iframe. Seems inconsistent with the rest of the app.